src/KeyEntityFormEnhancer.php

```diff
@@ -88,6 +89,13 @@ final class KeyEntityFormEnhancer {
    */
   private $configFactory;
 
+  /**
+   * The email validator.
+   *
+   * @var \Drupal\Component\Utility\EmailValidatorInterface
+   */
+  protected $emailValidator;
```

Comment:
Using `protected` in a `final` class does not make any sense.

Reply:
Changed.